autovalet_system/launch/state_machine.launch

```diff
@@ -0,0 +1,51 @@
+<launch>
+
+    <arg name="simulation" default="true"/>
+
+    <!-- SIMULATION -->
+    <group if="$(arg simulation)">
+
+        <!-- RTAB -->
+        <node name="SLAM" pkg="autovalet_system" type="slam.sh" args="true"/>
+
+        <!-- MOVEBASE -->
+        <node name="NAVIGATION" pkg="autovalet_system" type="nav.sh" args="true" />
```

Comment:
should the `true` rather be `$(arg simulation)`? looks more intuitive to me. I believe there are other lines with the same situation if you think fixing this is reasonable. (also in the hardware section)

Reply:
done